apps/dashboard/app/javascript/packs/files/file_ops.js

```diff
@@ -558,7 +565,9 @@ class FileOps {
   reportTransfer(data) {
     // 1. add the transfer label
     this.findAndUpdateTransferStatus(data);
-    this.poll(data);
+    let myFileOp = new FileOps();
+    // we pass myFileOp to the poller because we cannot use "this" inside the poller to reference the object.
+    this.poll(data, myFileOp);
```

Comment:
I don't think we should be passing the fileOps in here.  `poll` is a function on the fileOps class, so it's very weird to have to pass in a different object of the same type into it.

I wanted to figure this out too, so I did this where we use `this` & `that`.
https://github.com/OSC/ondemand/commit/e4d65eede4ed56f2972c86a6f87e9dfe8870d390

Reply:
@johrstrom This solution worked perfectly. Thank you!!!